### ISSUE TITLE
New argument parsing library is splitting remaining args on =, messing up URL launches

### ISSUE THREAD

**Issue by Aldaviva** [Aldaviva/VivaldiCustomLauncher#7]
1. Try to open a URL that includes query parameter key-value pairs, such as
    ```text
    http://www.jetbrains.com/resharperplatform/help?Wave=222&Keyword=Sharing_Configuration_Options#solution-personal-layer
    ```
1. `EntryPoint` splits the unrecognized argument on `=` into three separate arguments
    - `http://www.jetbrains.com/resharperplatform/help?Wave`
    - `222&Keyword`
    - `Sharing_Configuration_Options#solution-personal-layer`
1. VivaldiCustomLauncher starts Vivaldi with these three separate arguments, and Vivaldi opens 3 tabs, one for each argument

I don't know if there's a way to make EntryPoint stop doing this, or if I'm going to have to write my own argument parser for .NET (even though there are already 13 popular open-source argument parsing libraries. I can look at the other libraries in my .NET Command-Line Argument Parser Shootout and see if switching to `CommandLineParser`, `CommandLineUtils`, or `docopt` would work.

**Comment by Aldaviva** [comment]
It looks like the 3 alternative libraries should be able to handle this argument just fine:
```text
commandLineParser
vivaldi-application-directory = "C:\Programs\Internet\Vivaldi\Application", noVivaldiLaunch = True, extras = http://www.jetbrains.com/resharperplatform/help?Wave=222&Keyword=Sharing_Configuration_Options#solution-personal-layer

commandLineUtils
vivaldi-application-directory = "C:\Programs\Internet\Vivaldi\Application", noVivaldiLaunch = True, extras = http://www.jetbrains.com/resharperplatform/help?Wave=222&Keyword=Sharing_Configuration_Options#solution-personal-layer

docopt
vivaldi-application-directory = "C:\Programs\Internet\Vivaldi\Application", noVivaldiLaunch = True, extras = http://www.jetbrains.com/resharperplatform/help?Wave=222&Keyword=Sharing_Configuration_Options#solution-personal-layer

entryPoint
vivaldi-application-directory = C:\Programs\Internet\Vivaldi\Application, noVivaldiLaunch = True, extras = http://www.jetbrains.com/resharperplatform/help?Wave, 222&Keyword, Sharing_Configuration_Options#solution-personal-layer
```

One other difference is that they will preserve quotation marks around values, so I may have to strip them manually if they cause downstream problems.

**Comment by Aldaviva** [comment]
Should be fixed in [1.2.6](https://github.com/Aldaviva/VivaldiCustomLauncher/releases/tag/1.2.6) now.

**Comment by Aldaviva** [comment]
Moved from EntryPoint to [McMaster.Extensions.CommandLineUtils](https://www.nuget.org/packages/McMaster.Extensions.CommandLineUtils/), which is the only command line argument parser I've found for .NET that
- supports `--foo=bar` syntax
- keeps track of extra arguments, including extra `--options`, without splitting on `=`
- strips surrounding `"` characters from values

And I've tested 13 of them.
